cpp/ql/src/Security/CWE/CWE-704/WcharCharConversion.ql

```diff
@@ -0,0 +1,29 @@
+/**
+ * @name Cast between semantically different string types: char* from/to wchar_t*
+ * @description This rule indicates a potentially incorrect cast from/to an ANSI string (char *) to/from a Unicode string (wchar_t *). 
```

Comment:
The `@description` is out of date with the recent changes. For consistency with our other queries, I suggest rephrasing it as follows: "Casting a byte string to a wide-character string is likely to yield a string that is incorrectly terminated or aligned. This can lead to undefined behavior, including buffer overruns."

Reply:
Fixes ready.